x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/search_or_filter.cy.ts

```diff
@@ -25,13 +25,15 @@ import {
   showDataProviderQueryBuilder,
 } from '../../../tasks/timeline';
 import { waitForTimelinesPanelToBeLoaded } from '../../../tasks/timelines';
+import { deleteTimelines } from '../../../tasks/api_calls/common';
 
 import { hostsUrl, TIMELINES_URL } from '../../../urls/navigation';
 
 describe('Timeline search and filters', { tags: ['@ess', '@serverless'] }, () => {
   describe('timeline search or filter KQL bar', () => {
     beforeEach(() => {
       login();
+      deleteTimelines();
       visitWithTimeRange(hostsUrl('allHosts'));
     });
 
```

Comment:
let's rename the 2 following test to use the `should...` convention

Reply:
Done https://github.com/elastic/kibana/pull/173367/commits/581d66501d99ee01470ecf951263042c660b2310